migration/migration.go

```diff
@@ -429,3 +432,100 @@ func loadFields(ctx context.Context, wit *workitem.WorkItemType, into workitem.F
 
 	return nil
 }
+
+// migrates work item fields stored in a JSON document by converting the current 'system.description'
+// string element into an object with 2 fields: 'raw_value' and 'markup_type'  (related to issue #477)
+func migrateWorkItemDescriptionsToMarkup() fn {
```

Comment:
I'm almost 100% sure that this whole function can be done simpler in pure SQL. @aslakknutsen what do you think?